plugins/nodes/src/js/components/NodesGridView.tsx

```diff
@@ -10,6 +10,14 @@ import NodesGridDials from "./NodesGridDials";
 const MAX_SERVICES_TO_SHOW = 32;
 const OTHER_SERVICES_COLOR = 32;
 
+// This interface has just been crafted to reenable type checking.
+// It may very well be that it needs extension.
+interface Service {
```

Comment:
Shall we put it into `plugins/services/src/js/structs/Service.d.ts`? That was the way we started introducing types. Maybe there's a better way?

Reply:
With my understanding of them it would feel like a misbuse of `.d.ts`-files, as those are a tool to (partially) type libraries that would be hard to translate into pure `.ts`-files. From what i understood is that we want to convert app-code to TS in a boyscout-approach.

To get things done i just put it there, because i did not want to dig into structs just then.

I'd prefer to clean this up in a separate PR. Maybe we just remove that new declaration in favor of an `any[]` for now (although i don't like those).

Reply:
i removed the new interface for now. will type stuff in a spike day effort.